sdk/azidentity/TOKEN_CACHING.MD

```diff
@@ -0,0 +1,67 @@
+## Token caching in the Azure Identity client library
+
+*Token caching* is a feature provided by the Azure Identity library that allows apps to:
```

Comment:
Do we use the term "Azure Identity library" elsewhere or should it be "`azidentity` module"?  Not advocating for either, just as long as it's consistent.

Reply:
We use "Azure Identity client module" in the readme, so I'll use that here as well